index.html

```diff
@@ -0,0 +1,170 @@
+                            people from over 80 countries is taking place in October, in Korea.</p>
+                    </div>
+                    <h3>2015.10.15(THU) ~ 16(FRI)</h3>
+                    <h4>@ National Museum of Karea, Art Center Nabi and more</h4>
+                
+            </div>
+        </div>
+        <div class="main-program">
+            <div class="container">
+            <h1>Main Program</h1>
+            <img src="images/title_underline.svg" alt="underline">
+            <div class="prog-sec">
+                <div class="prog-items">
+                    <img src="images/Lecture.svg" alt="lecture">
+                    <h2>Lecture</h2>
+                    <p>Listen to the speakers from various contries about the messages of sharing and openning.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/CC.svg" alt="CC Excibition">
+                    <h2>CC Exhibition</h2>
+                    <p>Appreciate various creations applying CC license of artists, organized from art center Nabi.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/forum.svg" alt="Forum">
+                    <h2>Forum</h2>
+                    <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/workshop.svg" alt="Workshop">
+                    <h2>Workshop</h2>
+                    <p>Try creating your own work using open source license rather than just wathcing</p>
+                </div>
+
+                <div class="prog-items">
+                    <img src="images/ignite.svg" alt="CC Ignite">
+                    <h2>CC Ignite</h2>
+                    <p>get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                </div>
+            </div>
+            <a href="#">see the whole Program</a>
+            <button class="join">Join CC Global Summit 2015</button>
+        </div>
+        </div>
+
+        <div class="featured">
+            <div class="container">
+            <h1>Featured Speakers</h1>
+            <div class="featured-items">
+
+                <div class="featured-speakers">                    
+                </div>
+                <button class="more">more</button>
+                <div class="featured-speakers2"></div>
+                
+
+            </div>
+        </div>
+            <div class="partner">
+                <div class="container">
+                    <h6>Partner</h6>
+                <div class="grid3">
+                    <a href="#">mozilla</a>
+                    <a href="#">Google</a>
+                    <a href="#">Naver</a>
+                    <a href="#">daumkakao</a>
+                    <a href="#">airbnb</a>
+                </div>
+                </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement the [previous reviewer's suggestion](https://github.com/abdozayan12/capstone-project---Conference-page/tree/Conference---page#r1203932415), kindly add `horizontal` line under the `Partner` text 🙏🏼 

![image](https://github.com/abdozayan12/capstone-project---Conference-page/assets/63915024/b41cb6c7-526b-4b67-833e-0814f99b2b8e)